director/release_archive_with_metadata.go

```diff
@@ -2,7 +2,7 @@ package director
 
 import (
 	"archive/tar"
-	"compress/gzip"
+	gzip "github.com/klauspost/pgzip"
```

Comment:
same as in `bosh-utils`: this import alias only hides info, would recommend removing it.

also putting non-standard-lib imports here will make the linter 😡